gherkin/flagd-json-evaluator.feature

```diff
@@ -30,6 +30,15 @@ Feature: flagd json evaluation
       | "nine"  | "hearts"   |
       | 3       | "diamonds" |
 
+  Scenario Outline: Fractional operator shorthand
+    When a string flag with key "fractional-flag-shorthand" is evaluated with default value "fallback"
+    And a context containing a targeting key with value <targeting key>
+    Then the returned value should be <value>
```

Comment:
These bindings already exist, so no new implementation code needs to be written. Consumers can just update the flagd image and the submodule containing the gherkin.